### PR TITLE
Restructure CD cronjobs

### PR DESCRIPTION
**Pull request by afrittoli**
# Changes

Tekton CD is based on a number of cronjobs that trigger Tekton
template triggers to perform various CD functions.
Organise cronjobs per cluster and per function, and improve the
documentation about cronjobs by adding links to the underlying
template triggers.

In the process of moving files around, rename several folders
to simple and non-redundant names. The target cronjob names
will not change as part of this process.

This work is required to be able to CD the cronjobs themselves,
since we'll need one job per cluster that runs cronjobs.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md)
for more details._

/kind cleanup
/cc @vdemeester @bobcatfish

### CODE REVIEW

File: tekton/cronjobs/README.md [hunk 1]
```diff
@@ -1,93 +1,107 @@
...
 
 This folder holds kustomize overlays, that are used to maintain cron job
-configurations. To add a new cron job to be deployed to the `dogfooding`
-cluster, create a folder and add a `kustomization.yaml` into it, along with the
-cronjob overlay.
-
-There are three base cron jobs available:
-* `nightly-image-build-cron-base` which can be used to build container images
-  on a regular basis and push them to a container repo (by default
+configurations. To add a new cron job to be deployed to the a cluster,
```
Comment (sbwsg) [inline tekton/cronjobs/README.md:4]: nit: `to the a cluster` -> `to a cluster`

File: tekton/cronjobs/dogfooding/configmaps/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+resources:
+- labels-sync-hourly
+- prow-labels-sync-hourly
+- prow-config-hourly
+- prow-pluings-hourly
```
Comment (sbwsg) [inline tekton/cronjobs/dogfooding/configmaps/kustomization.yaml:5]: should this be `prow-plugins-hourly`?

Comment (afrittoli) [reply tekton/cronjobs/dogfooding/configmaps/kustomization.yaml:5]: Good catch, thanks. The name of the folder was also `prow-pluings-hourly` which is why I did not catch this when testing the build :)

File: tekton/cronjobs/README.md (hunk 1 above)
Comment (afrittoli) [reply tekton/cronjobs/README.md:4]: done

**Pull request merged**